src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/VanillaMachinesListener.java

```diff
@@ -97,6 +97,20 @@ public void onAnvil(InventoryClickEvent e) {
         }
     }
 
+    @EventHandler(ignoreCancelled = true)
+    public void onCartographyTable(InventoryClickEvent e) {
+        if (e.getRawSlot() == 2 && e.getInventory().getType() == InventoryType.CARTOGRAPHY
```

Comment:
The Cartography table doesn't exist in 1.13 I think, so this will require a version check here